src/components/UI/Routines.js.jsx

```diff
@@ -1,12 +1,30 @@
 import React, { Component } from 'react'
+import { Link } from 'react-router'
 import { Col } from 'react-bootstrap'
+import Routine from './Routine.js.jsx'
 import './styles/Routines.css'
 
 class Routines extends Component {
+  componentDidMount() {
+    this.getRoutines(8)
+  }
+
+  getRoutines = (num) => {
```

Comment:
@pwentz the reason we have `num`, is because we would only show x number of routines/exercises in the component? Are you getting the routines from another react component (I'm not 100% sure of what's going on).